_es/tour/basics.md

```diff
@@ -6,4 +6,319 @@ discourse: false
 
 partof: scala-tour
 language: es
+
+num: 2
+next-page: unified-types
+previous-page: tour-of-scala
 ---
+
+En esta página, cubriremos lo básico de Scala.
```

Comment:
```suggestion
En esta página, practicaremos conceptos básicos de Scala.
```

I think the fact that the page is `trying scala in the browser` is inviting users to practice rather than _just_ read. The suggestion reads: "In this page, we'll practice basic scala concepts"